Example/Example/ContentView.swift

```diff
@@ -61,15 +60,32 @@ struct ContentView: View {
                 Text("Hide/Show TabBar")
             }
             .tabItem(for: Item.first)
-            
-            Text("Second")
+
+            TextWrapper()
                 .tabItem(for: Item.second)
-            
-            Text("Third")
+
+            TextWrapper()
                 .tabItem(for: Item.third)
         }
         .tabBar(style: CustomTabBarStyle())
         .tabItem(style: CustomTabItemStyle())
+        .onChange(of: selection) { newValue in
+            print("selection changed:", newValue)
+        }
```

Comment:
I think we should remove this `print` statement, as it doesn't add any significant value here.

Reply:
Just for the purpose of testing, can remove it.

Reply:
Yes, remove it please.